kedro-datasets/kedro_datasets/matplotlib/matplotlib_writer.py

```diff
@@ -73,12 +74,15 @@ class MatplotlibWriter(
         >>> from kedro_datasets.matplotlib import MatplotlibWriter
         >>>
         >>> plots_dict = {}
-        >>> for colour in ["blue", "green", "red"]:
+        >>> for colour in ["blue", "green", "red"]:  # doctest: +ELLIPSIS
```

Comment:
```suggestion
        >>> for colour in ["blue", "green", "red"]:
```
Seems this may not be necessary for some reason.